### ISSUE TITLE
The new logo may not be suitable

### ISSUE THREAD

**Issue by reaCodes** [Microsoft/vscode-loc#19]
When the background is black, a part of the logo is visible.

---

![qq 20180506145538](https://user-images.githubusercontent.com/13817254/39670746-0f34e102-513e-11e8-94f1-17d1c58d8999.png)
![qq 20180506150004](https://user-images.githubusercontent.com/13817254/39670753-34fff7dc-513e-11e8-8737-2a48f4b60529.png)

**Comment by levrik** [comment]
![image](https://user-images.githubusercontent.com/9491603/39671452-d839ab44-5118-11e8-8f8c-04fd7e027d9a.png)
Looks fine for the German language pack.

**Comment by reaCodes** [comment]
@levrik Because these two logos are not same.

**Comment by levrik** [comment]
@reaCodes Yeah, I'm not dumb 😄 Just thought that it maybe got fixed already.
